packages/office-ui-fabric-react/src/components/Icon/Icon.base.tsx

```diff
@@ -22,12 +22,13 @@ export class IconBase extends BaseComponent<IIconProps, IIconState> {
   }
 
   public render() {
-    const { ariaLabel, className, styles, iconName, imageErrorAs } = this.props;
+    const { ariaLabel, className, styles, theme, iconName, imageErrorAs } = this.props;
     const isPlaceholder = typeof iconName === 'string' && iconName.length === 0;
     const isImage = this.props.iconType === IconType.image || this.props.iconType === IconType.Image;
     const { iconClassName, children } = this._getIconContent(iconName);
 
     const classNames = getClassNames(styles, {
+      theme: theme!,
```

Comment:
this won't throw on initial render when it tries to deconstruct theme?

Reply:
Removing change to icon due to its irrelevance to this PR.  Change was originally to affect IconButton which turned out to be part of button